### PR TITLE
MDSK:  Add wraith into the VM

### PR DESCRIPTION
**Pull request by rigoucr**
Just follow the instructions in the readme file for the generated project.

### CODE REVIEW

File: generators/app/templates/7/_README.md [hunk 1]
```diff
@@ -26,43 +26,16 @@ vagrant plugin install vagrant-auto_network
 ```
 
 ### Wraith
-In this generator we include Wraith for visual regression testing so you'll need the following in order to be able to use it:
+In this generator we include Wraith for visual regression testing,  to use it go to `wraith/configs/capture.yaml` and change the domains urls, the paths you want to test and the breakpoints if needed, then go back to the root path of your project and run:
```
Comment (kporras07) [inline generators/app/templates/7/_README.md]: Double space after comma.

File: generators/app/templates/7/scripts/wraith_testing.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```
Comment (kporras07) [inline generators/app/templates/7/scripts/wraith_testing.sh]: For standards compliment, could you name this file _wraith_testing? Then, ensure it get the right name on the generated project.

File: generators/app/templates/8/_README.md [hunk 3]
```diff
@@ -26,43 +26,16 @@ vagrant plugin install vagrant-auto_network
 ```
 
 ### Wraith
-In this generator we include Wraith for visual regression testing so you'll need the following in order to be able to use it:
+In this generator we include Wraith for visual regression testing,  to use it go to `wraith/configs/capture.yaml` and change the domains urls, the paths you want to test and the breakpoints if needed, then go back to the root path of your project and run:
```
Comment (kporras07) [inline generators/app/templates/8/_README.md]: Same here.

File: generators/app/templates/7/provisioning/tasks/wraith.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- name: Install phantomjs for wraith.
+  npm:
+    name: phantomjs
+    global: yes
+    
+- name: Add Ruby gems to the Path.
+  lineinfile:
+    dest: /home/vagrant/.bash_profile
+    create: yes
+    line: 'export PATH="$HOME/.gem/ruby/2.3.0/bin:$PATH"'
```
Comment (kporras07) [inline generators/app/templates/7/provisioning/tasks/wraith.yml:23]: How are you sure you're getting ruby 2.3.0?

File: generators/app/templates/8/provisioning/tasks/wraith.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+- name: Install phantomjs for wraith.
+  npm:
+    name: phantomjs
+    global: yes
+    
+- name: Add Ruby gems to the Path.
+  lineinfile:
+    dest: /home/vagrant/.bash_profile
+    create: yes
+    line: 'export PATH="$HOME/.gem/ruby/2.3.0/bin:$PATH"'
```
Comment (kporras07) [inline generators/app/templates/8/provisioning/tasks/wraith.yml:23]: Same here

File: generators/app/templates/8/scripts/wraith_testing.sh [hunk 6]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```
Comment (kporras07) [inline generators/app/templates/8/scripts/wraith_testing.sh]: Same here

**Comment by kporras07** [comment]
It looks awesome! Only a couple of minor changes.

File: generators/app/templates/7/provisioning/tasks/wraith.yml (hunk 4 above)
Comment (rigoucr) [reply generators/app/templates/7/provisioning/tasks/wraith.yml:23]: @kporras07 I do some changes and now I specify the ruby version to be installed in the default.config.yml

File: generators/app/templates/8/provisioning/tasks/wraith.yml (hunk 5 above)
Comment (rigoucr) [reply generators/app/templates/8/provisioning/tasks/wraith.yml:23]: @kporras07 I do some changes and now I specify the ruby version to be installed in the default.config.yml

**Comment by kporras07** [comment]
@rigoucr :shipit:

**Pull request merged**